modules/embree/4.3.0/patches/bazel_build.patch

```diff
@@ -0,0 +1,658 @@
+--- BUILD.bazel
++++ BUILD.bazel
+@@ -0,0 +1,655 @@
++"""
++    SPDX-FileCopyrightText: 2023 Julian Amann <dev@vertexwahn.de>
++    SPDX-License-Identifier: Apache-2.0
++"""
++
++package(
++    default_visibility = ["//visibility:public"],
++)
++
++config_setting(
++    name = "windows_x86_64",
++    values = {"cpu": "x64_windows"},
++    visibility = ["//visibility:public"],
++)
++
++load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
++
++# If you want to use the internal tasking system use:
++# bazel build //... --define tasking_system=internal
```

Comment:
Could we namespace this define with `embree`? Alternatively, you could use a custom Starlark flag from skylib.

Reply:
Added embree_ prefix